cms-instructions.md

```diff
@@ -0,0 +1,33 @@
+## Contractor Management System Instructions
```

Comment:
General comment for whole file.  Would prefer that md files still adhere to the 80 column limit when possible (not always possible with links and such).   This would be consistent with the `README.md` formatting and it makes it easier to edit when the text is not one long line.